docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx

```diff
@@ -32,13 +32,13 @@ The public key of the issuing account is linked on the ledger to the asset. Resp
 
 In Horizon, assets are represented in a JSON object:
 
-```json5
```

Comment:
Comments don't render with `json` thus `json5` which does support comments

Reply:
Yeah we really only need those blocks if there's a list of different code languages otherwise it's a list of examples with one example